nevergrad/optimization/optimizerlib.py

```diff
@@ -417,13 +423,14 @@ def es(self) -> tp.Any:  # typing not possible since cmaes not imported :(
                     seed=np.nan,
                     CMA_elitist=self._config.elitist,
                 )
-
+                if self._normalizer is not None:
+                    inopts["bounds"] = [0, 1]
                 inopts.update(self._config.inopts if self._config.inopts is not None else {})
                 self._es = cma.CMAEvolutionStrategy(
                     x0=self.parametrization.sample().get_standardized_data(reference=self.parametrization)
                     if self._config.random_init
                     else np.zeros(self.dimension, dtype=np.float_),
-                    sigma0=self._config.scale,
+                    sigma0=self._config.scale if self._normalizer is None else 0.3,
```

Comment:
this may be unexpected by a user, should we set scale to None as default, and if None then the scale will be 1.0 if not normalized, else 0.3?
This way if the user provides a scale, it is actually used? Like in Mini/MilliCMA etc... otherwise they are all equivalent